src/app.cpp

```diff
@@ -216,7 +216,13 @@ int main(int argc, char* argv[]) {
 	}
 	else if (magic == PCAP_MAGIC_NUMBER || magic == PCAP_MAGIC_NUMBER_LITTLE_ENDIAN) {
 		char errbuf[PCAP_ERRBUF_SIZE];
-		pcap_t* infile = pcap_open_offline(args::get(inarg).c_str(), errbuf);
+
+		pcap_t* infile;
+#if defined(PCAP_TSTAMP_PRECISION_NANO)
```

Comment:
Current release in CI is compiled with an old Npcap version, so this will never be true in auto generated releases